### PR TITLE
section: Wallet Checking the Non-Revocation of its Wallet Provider

### PR DESCRIPTION
**Pull request by peppelinux**
This PR replaces a placeholder about the section related to the trust establishment made by wallet instance to its wallet provider.

### CODE REVIEW

File: openid-federation-wallet-1_0.md [hunk 1]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
...
+
+The process to discover the trust with a Wallet Provider is equivalent to the one made for discoving the trust with a Credential Issuer, as described in the dedicated section below.
 
 ## Wallet Discovering Credentials Issuers
 
 Wallets begin by discovering the identity of Credential Issuers through the federation's trust infrastructure. This involves retrieving the Credential Issuer's Entity Configuration and verifying its Trust Chain up to a recognized Trust Anchor. The Credential Issuer’s Entity Configuration provides essential information, including its roles within the federation, policies it adheres to, and cryptographic keys for secure communication.
 
-In the example represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
+the process described in the  represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:466]:
```suggestion
the process described in the represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
```

File: openid-federation-wallet-1_0.md [hunk 2]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
 
 ## Wallet Checking the Non-Revocation of its Wallet Provider
 
-...
+Wallets SHOULD periodically check their Wallet Providers compliance through the federation's trust infrastructure. This involves retrieving the Wallet Provider's Entity Configuration and verifying its Trust Chain up to a recognized Trust Anchor, ensuring that the Wallet Provider has not been revoked within the federation. Wallets SHOULD remain neutral in attesting to the reliability of their Wallet Providers for the End-User, thereby protecting the End-User against any malevolent behavior by the Wallet Provider.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:456]:
```suggestion
Wallets SHOULD periodically check their Wallet Providers' compliance through the federation's trust infrastructure. This involves retrieving the Wallet Provider's Entity Configuration and verifying its Trust Chain up to a recognized Trust Anchor, ensuring that the Wallet Provider has not been revoked within the federation. Wallets SHOULD remain neutral in attesting to the reliability of their Wallet Providers for the End-User, thereby protecting the End-User against any malevolent behavior by the Wallet Provider.
```

File: openid-federation-wallet-1_0.md [hunk 3]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
...
+
+The Wallet Provider’s Entity Configuration provides essential information, including its roles within the federation, policies it adheres to, and cryptographic keys for secure communication. In the example represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Wallet Providers enabled within the federation.
+
+The process to discover the trust with a Wallet Provider is equivalent to the one made for discoving the trust with a Credential Issuer, as described in the dedicated section below.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:460]:
```suggestion
The process to discover the trust with a Wallet Provider is equivalent to the one used for discoving the trust with a Credential Issuer, as described in the dedicated section below.
```

File: openid-federation-wallet-1_0.md [hunk 4]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
...
 
 ## Wallet Discovering Credentials Issuers
 
 Wallets begin by discovering the identity of Credential Issuers through the federation's trust infrastructure. This involves retrieving the Credential Issuer's Entity Configuration and verifying its Trust Chain up to a recognized Trust Anchor. The Credential Issuer’s Entity Configuration provides essential information, including its roles within the federation, policies it adheres to, and cryptographic keys for secure communication.
 
-In the example represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
+the process described in the represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:466]:
```suggestion
In the process represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Credential Issuers enabled within the federation.
```

**Review by selfissued — APPROVED** [review]
Approved with suggestions

File: openid-federation-wallet-1_0.md [hunk 5]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
 
 ## Wallet Checking the Non-Revocation of its Wallet Provider
 
-...
+Wallets SHOULD periodically check their Wallet Providers' compliance through the federation's trust infrastructure. This involves retrieving the Wallet Provider's Entity Configuration and verifying its Trust Chain up to a recognized Trust Anchor, ensuring that the Wallet Provider has not been revoked within the federation. Wallets SHOULD remain neutral in attesting to the reliability of their Wallet Providers for the End-User, thereby protecting the End-User against any malevolent behavior by the Wallet Provider.
+
+The Wallet Provider’s Entity Configuration provides essential information, including its roles within the federation, policies it adheres to, and cryptographic keys for secure communication. In the example represented in the sequence diagram below, the Wallet Instance uses the Federation API to discover and collect all the Wallet Providers enabled within the federation.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:458]:
```suggestion
The Wallet Provider’s Entity Configuration provides essential information, including its roles within the federation, policies it adheres to, and cryptographic keys for secure communication. The Wallet Instance SHOULD use the Federation API to periodically reestablish trust with its Wallet Provider.
```

File: openid-federation-wallet-1_0.md [hunk 6]
```diff
@@ -455,13 +453,17 @@ The process of trust establishment in federated environments is illustrated in t
...
+
+The process to discover the trust with a Wallet Provider is equivalent to the one used for discoving the trust with a Credential Issuer, as described in the dedicated section below.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:460]:
```suggestion
The process to discover the trust with a Wallet Provider is equivalent to the one used for discovering the trust with a Credential Issuer, as described in the dedicated section below.
```

**Pull request merged**